dockerfiles/3.3.6-browsers/Dockerfile

```diff
@@ -8,13 +8,13 @@
 # https://medium.com/@tariq.m.islam/container-deployments-a-lesson-in-deterministic-ops-a4a467b14a03
 # You can get the hash value for a specific image by using "docker images"
 # and querying about REPOSITORY:TAG, for example:
-# docker pull cimg/ruby:3.2.2-browsers
+# docker pull cimg/ruby:3.3.6-browsers
```

Comment:
These were only commented examples, but sure!

Reply:
Ah yeah, I had an old checkout locally and I did this before I noticed it was already updated (thus the branch name also)